src/hotspot/share/gc/shared/collectedHeap.cpp

```diff
@@ -274,6 +274,10 @@ CollectedHeap::CollectedHeap() :
     _perf_gc_lastcause =
                 PerfDataManager::create_string_variable(SUN_GC, "lastCause",
                              80, GCCause::to_string(_gc_lastcause), CHECK);
+
+    _perf_parallel_gc_threads_cpu_time =
+                PerfDataManager::create_variable(NULL_NS, "par_gc_thread_time",
```

Comment:
See my general comment about name spaces. The name should be something like `*.cpu_time`.